packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -77,6 +77,10 @@ export type SuiteAction =
           type: typeof SUITE.SET_THEME;
           variant: SuiteThemeVariant;
           colors: SuiteThemeColors;
+      }
+    | {
+          type: typeof SUITE.SET_AUTODETECT;
+          payload: Partial<AutodetectSettings>;
```

Comment:
What about removing `Partial` and set attributes in `AutodetectSettings` to optional?

Reply:
Why? Considered `AutodetectSettings` is part of application's global state and defaults to `{ language: true, theme: true }`, I don't think it makes sense to be able to set it back to undefined values.